packages/react-dom/src/events/EventSystemFlags.js

```diff
@@ -25,4 +25,4 @@ export const SHOULD_NOT_DEFER_CLICK_FOR_FB_SUPPORT_MODE =
 // will result in endless cycles like an infinite loop.
 // We also don't want to defer during event replaying.
 export const SHOULD_NOT_PROCESS_POLYFILL_EVENT_PLUGINS =
-  IS_EVENT_HANDLE_NON_MANAGED_NODE | IS_NON_DELEGATED | IS_CAPTURE_PHASE;
+  IS_EVENT_HANDLE_NON_MANAGED_NODE | IS_NON_DELEGATED;
```

Comment:
I'm a little concerned based off this comment here:
https://github.com/facebook/react/blob/534c9c52ec3e33ed241f7a20e264bb4571d452b3/packages/react-dom/src/events/DOMPluginEventSystem.js#L120

Not sure what the effect on those plugins would be or how to test they dont act up? @gaearon  any suggestions?

Reply:
hmm I see some failing tests I'll take a look at those first